### PR TITLE
Update prod scripts

### PR DESCRIPTION
**Pull request by JavaDeveloper456788**

### CODE REVIEW

File: .github/workflows/deploy_to_prod.yml [hunk 1]
```diff
@@ -96,20 +105,20 @@ jobs:
           oc tag ${{ env.OPENSHIFT_NAMESPACE_TEST }}/${{ env.REPO_NAME }}-cas-api:${{ steps.get-latest-tag.outputs.tag }} \
           ${{ env.OPENSHIFT_NAMESPACE_PROD }}/${{ env.REPO_NAME }}-cas-api:${{ steps.get-latest-tag.outputs.tag }}
 
+          curl -s https://raw.githubusercontent.com/bcgov/${{ env.REPO_NAME }}/${{ env.BRANCH }}/openshift/update-configmap.sh | bash /dev/stdin  "${{ env.REPO_NAME }}" "${{ env.OPENSHIFT_NAMESPACE_DEV }}" "${{env.APP_DEBUG}}" "${{env.DOCUMENTROOT}}" "${{ env.DYNAMICSBASEURL}}" "${{env.ASPNETCORE_ENVIRONMENT}}" "${{env.DYNAMICSAUTHENTICATIONSETTINGS__ACTIVEENVIRONMENT}}" "${{env.DYNAMICSAUTHENTICATIONSETTINGS__CLOUDWEBAPIURL}}" "${{env.DYNAMICSAUTHENTICATIONSETTINGS__CLOUDREDIRECTURL}}" "${{env.DYNAMICSAUTHENTICATIONSETTINGS__CLOUDRESOURCEURL}}" "${{env.DYNAMICSAUTHENTICATIONSETTINGS__CLOUDBASEURL}}"  "${{env.ASPNETCORE_ENVIRONMENT_CAS}}"      "${{env.CAS_API_SERVER}}"  "${{env.CAS_TOKEN_URI}}"  "${{env.CAS_INVOICE_URI}}"
```
Comment (cditcher) [inline .github/workflows/deploy_to_prod.yml:108]: The second argument for update-configmap.sh (${{env.OPENSHIFT_NAMESPACE_DEV}}) is not used in the script and also the environment variable does not exist. Suggest removing this argument from the script and updating other workflows that use this argument also.

**Review by cditcher — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_to_prod.yml (hunk 1 above)
Comment (JavaDeveloper456788) [reply .github/workflows/deploy_to_prod.yml:108]: Fixed

File: .github/workflows/deploy_to_prod.yml
Comment (cditcher) [inline .github/workflows/deploy_to_prod.yml:1]: Please also update checkout to v3 and also remove additional checkout on the deploy step.

**Review by cditcher — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_to_prod.yml (hunk 1 above)
Comment (JavaDeveloper456788) [reply .github/workflows/deploy_to_prod.yml]: I Apologize. I removed this from test and forgot to do the same with prod. Fixed now.

File: .github/workflows/deploy_test.yml
[the remark this answers is not in the document]
Comment (JavaDeveloper456788) [reply .github/workflows/deploy_test.yml:1]: Fixed

File: .github/workflows/build.from.master.branch.deploy.to.dev.yml
[the remark this answers is not in the document]
Comment (JavaDeveloper456788) [reply .github/workflows/build.from.master.branch.deploy.to.dev.yml:1]: Fixed

File: .github/workflows/deploy_test.yml
[the remark this answers is not in the document]
Comment (JavaDeveloper456788) [reply .github/workflows/deploy_test.yml:1]: Fixed

File: .github/workflows/deploy_to_prod.yml
Comment (JavaDeveloper456788) [reply .github/workflows/deploy_to_prod.yml:1]: Fixed

**Review by cditcher — APPROVED** [review]

**Pull request merged**